Modules/TOF/CMakeLists.txt

```diff
@@ -112,6 +112,7 @@ install(FILES tofdigits.json
               tofMatchedTracks_ITSTPCTOF_TPCTOF_MC.json
               tofMatchedTracks_ITSTPCTOF_TPCTOF_direct.json
               tofMatchedTracks_ITSTPCTOF_TPCTOF_direct_MC.json
+	      tofMatchedTracks_ITSTPCTOF_TPCTOF_AllTypes_direct_MC.json
```

Comment:
Ciao @chiarazampolli could it be that this file was not committed? or that it should be renamed to/from `tofMatchedTracks_AllTypes_direct_MC.json`?

Reply:
I did fix it a while ago:

```
[O2PDPSuite/latest-o2] ~/SOFT/alibuild/ali-o2-dev/QualityControl $> git diff
diff --git a/Modules/TOF/CMakeLists.txt b/Modules/TOF/CMakeLists.txt
index a8dee66d..343a2e50 100644
--- a/Modules/TOF/CMakeLists.txt
+++ b/Modules/TOF/CMakeLists.txt
@@ -112,7 +112,7 @@ install(FILES tofdigits.json
               tofMatchedTracks_ITSTPCTOF_TPCTOF_MC.json
               tofMatchedTracks_ITSTPCTOF_TPCTOF_direct.json
               tofMatchedTracks_ITSTPCTOF_TPCTOF_direct_MC.json
-             tofMatchedTracks_ITSTPCTOF_TPCTOF_ITSTPCTRDTOF_direct_MC.json
+             tofMatchedTracks_ITSTPCTOF_TPCTOF_AllTypes_direct_MC.json
         DESTINATION etc)
 
 get_property(dirs
[O2PDPSuite/latest-o2] ~/SOFT/alibuild/ali-o2-dev/QualityControl $> git add Modules/TOF/CMakeLists.txt
[O2PDPSuite/latest-o2] ~/SOFT/alibuild/ali-o2-dev/QualityControl $> git commit -m "forgot to change name of file in CMakeLists"
[QCtofTrackTypes 8a9a6f34] forgot to change name of file in CMakeLists
 1 file changed, 1 insertion(+), 1 deletion(-)
[O2PDPSuite/latest-o2] ~/SOFT/alibuild/ali-o2-dev/QualityControl $> git rebase -i upstream/master
Successfully rebased and updated refs/heads/QCtofTrackTypes.
[O2PDPSuite/latest-o2] ~/SOFT/alibuild/ali-o2-dev/QualityControl $> git push -f origin HEAD:updateQCtof
Enumerating objects: 20, done.
Counting objects: 100% (20/20), done.
Delta compression using up to 32 threads
Compressing objects: 100% (11/11), done.
Writing objects: 100% (11/11), 2.15 KiB | 2.15 MiB/s, done.
Total 11 (delta 8), reused 0 (delta 0)
remote: Resolving deltas: 100% (8/8), completed with 8 local objects.
To github.com:chiarazampolli/QualityControl.git
 + 826f03a0...759a0f2a HEAD -> updateQCtof (forced update)
[O2PDPSuite/latest-o2] ~/SOFT/alibuild/ali-o2-dev/QualityControl $> 
```
I have to check why it was not propagated.

Reply:
It is there:
https://github.com/AliceO2Group/QualityControl/compare/826f03a0e3fc83902605649368a8255176d660a7..759a0f2a05f0a64637d47e4c4132af0f9192371f

Reply:
Found it, I made a typo.